src/DisplayTotalMasteryPage/DisplayTotalMasteryPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Container, makeStyles, Theme, Typography } from '@material-ui/core'
+import React, { FC, useContext, useState } from 'react'
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { GlobalContext } from '../GlobalContext'
+import { theme } from '../theme'
+
+interface DisplayTotalMasteryPageProps {}
+
+const useStyles = makeStyles({
+	resultRow: (theme: Theme) => ({
+		alignSelf: 'stretch',
+		border: '1px solid',
+		borderCollapse: 'collapse',
+		borderColor: theme.palette.primary.main,
+		display: 'flex',
+		justifyContent: 'space-between',
+		marginBottom: theme.spacing(3),
+	}),
+})
+
+const DisplayTotalMasteryPage: FC<DisplayTotalMasteryPageProps> = ({}) => {
+	const { name } = useParams<{ name: string }>()
+	const { api, setTitle } = useContext(GlobalContext) // for page title and API access
+	const classes = useStyles(theme)
+
+	const [result, setResult] = useState({
+		icon: '',
+		mastery: 0,
+		name: '',
+		summonerId: '',
+	})
+
+	const fireGetMastery = async (): Promise<void> => {
+		const summonerResult = await api.pingUserSearchEndpoint(name)
+		const masteryResult = await api.pingMasteryTotalEndpoint(
+			summonerResult.id
+		)
+		setResult({
+			icon: summonerResult.profileIconId,
+			mastery: masteryResult,
+			name: summonerResult.name,
+			summonerId: summonerResult.id,
+		})
+	}
+
+	useEffect(() => {
+		setTitle(`${name}'s mastery`)
+		fireGetMastery()
+	}, [])
+
+	return (
+		// Display user icon, username, and total mastery
+		<Container className="user-data-container">
+			<Typography
+				align="center"
+				color="primary"
+				gutterBottom
+				variant="h5"
```

Comment:
So sayeth @Roughnick and the accessibility guidelines, each page should have at least one h1 element; Nav has it (wrapping the title), so this should be the next level down (h2)

![image](https://user-images.githubusercontent.com/2125913/125541549-9694a097-ab4a-4766-9e16-efc06dd1a74c.png)

```suggestion
				variant="h2"
```